src/aws/bin/terraform

```diff
@@ -56,22 +56,22 @@ function terraform() {
         -u "$(id -u)" \
         -v "${PWD}/:/app" \
         -w "/app" \
-        --env-file ./env.d/development \
+        --env-file ./env.d/${MARSHA_TERRAFORM_ENV_FILE} \
         "hashicorp/terraform:0.14.5" \
         "$@"
     else
         docker run --rm -it \
         -u "$(id -u)" \
         -v "${PWD}/:/app" \
         -w "/app" \
-        --env-file ./env.d/development \
+        --env-file ./env.d/${MARSHA_TERRAFORM_ENV_FILE} \
         -e TF_VAR_marsha_base_url="${NGROK_URL}" \
         "hashicorp/terraform:0.14.5" \
         "$@"
     fi
 }
 
-OPTS=$(getopt -o "e" --long "env:" -- "$@")
+OPTS=$(getopt -o "e:" --long "env:" -- "$@")
```

Comment:
I don't know how this work.
I'll be grateful if you can explain it to me :)

Reply:
This option can have an argument `-e staging` for example. To instruct the function that this option expect an argument you put `:` after the option.